server/sonar-server/src/test/java/org/sonar/server/qualityprofile/ws/QProfilesWsTest.java

```diff
@@ -172,7 +172,7 @@ public void define_set_default_action() {
     WebService.Action setDefault = controller.action("set_default");
     assertThat(setDefault).isNotNull();
     assertThat(setDefault.isPost()).isTrue();
-    assertThat(setDefault.params()).hasSize(3);
+    assertThat(setDefault.params()).hasSize(4);
```

Comment:
no assertion on new param
this test will still succeed if the param is replaced by another one, whether on purpose or by mistake, this isn't right imho